drivers/extend/gpio_ext.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2018 Acutam Automation, LLC
+ *               2019 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     drivers_extend_gpio
+ *
+ * @{
+ *
+ * @file
+ * @brief       GPIO extension functions
+ *
+ * @author      Matthew Blue <matthew.blue.neuro@gmail.com>
+ * @author      Gunar Schorcht <gunar@schorcht.net>
+ *
+ * @}
+ */
+
+#if MODULE_EXTEND_GPIO
```

Comment:
Shouldn't this be `#ifdef` instead of `#if`?

Reply:
Since enabled modules lead to defintions like
```
#define MODULE_EXTEND_GPIO 1
```
in `<app>/bin/<board>/riotbuild/riotbuild.h`, the simple `#if` works as well.

Reply:
I was more worried about the case the module is not used, in which `MODULE_EXTEND_GPIO` is not defined